### PR TITLE
Add research org survey questions.

### PR DESCRIPTION
**Pull request by mrbobbytables**
For general review.

Questions have been adapted from the meeting notes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+The CNCF Research User Group’s purpose is to function as a focal point for the
+discussion and advancement of Research Computing using “Cloud Native”
+technologies. This includes enumerating current practices, identifying gaps,
+and directing effort to improve the Research Cloud Computing ecosystem.
 
-Agenda: https://docs.google.com/document/d/1vvXxW4Cd4P5gcmWGz-_yKbgJex2_NlSWaHtsk_56TnA/edit?ts=5d53c5ff#
+Twice-monthly meetings on the [second Wednesdays at 09:00 UTC][1], and the
+[fourth Wednesdays at 15:00 UTC][2].
+
+- [Meeting Agenda and Notes][Agenda]
```
Comment (k82cn) [inline README.md]: how about add links, e.g.  [Meeting Agenda and Notes]( https://docs.google.com/document/d/1vvXxW4Cd4P5gcmWGz-_yKbgJex2_NlSWaHtsk_56TnA/edit?ts=5d53c5ff# )

Comment (mrbobbytables) [reply README.md]: Fixed! Thanks for catching that. I thought I had done that with a reference link via `[Agenda]` 🤦‍♂

Comment (k82cn) [reply README.md]: lgtm, thanks :)

**Comment by mrbobbytables** [comment]
Going to go ahead and merge this in, thanks for the comments :)

**Pull request merged**